README.md

```diff
@@ -34,23 +34,23 @@ ArcGIS products give you access to APIs, location services, and tools to develop
 
 ## APIs and SDKs
 
-- [ArcGIS API for JavaScript](https://developers.arcgis.com/javascript/latest/) - Guides, sample code, API references and showcase to build 2D and 3D interactive web apps unlocking geospatial data.
 - [ArcGIS API for Python](https://developers.arcgis.com/python/) - Guides, sample notebooks and API reference to do mapping, spatial analysis, data science, geospatial AI and automation using Python.
 - [ArcGIS CityEngine SDKs (C++)](https://github.com/esri/cityengine-sdk) - C++ APIs, documentation and examples for the Procedural Runtime (PRT).
 - [ArcGIS CityEngine SDKs (Python)](https://github.com/Esri/pyprt) - Python bindings for the "Procedural Runtime" (PRT) of CityEngine.
 - [ArcGIS Earth Automation API](https://doc.arcgis.com/en/arcgis-earth/automation-api/get-started.htm) - Guide, API reference and samples to communicate with ArcGIS Earth.
 - [ArcGIS Enterprise SDK](https://developers.arcgis.com/enterprise-sdk/) - Guides, API reference and sample code to extent ArcGIS Enterprise.
+- [ArcGIS Maps SDK for .NET](https://developers.arcgis.com/net/) - Guides, sample codes and API reference to build desktop and mobile apps using .Net.
+- [ArcGIS Maps SDK for Android](https://developers.arcgis.com/android/) - Guides, sample codes and API reference to build mobile apps using Java or Kotlin.
+- [ArcGIS Maps SDK for iOS](https://developers.arcgis.com/ios/) - Guides, sample codes and API reference to build mobile apps using Swift or Objective-C.
```

Comment:
These have new product names. Might have to reorder as well
```suggestion
- [ArcGIS Maps SDK for Kotlin](https://developers.arcgis.com/kotlin/) - Guides, sample codes and API reference to build mobile apps using Kotlin.
- [ArcGIS Maps SDK for Swift](https://developers.arcgis.com/swift/) - Guides, sample codes and API reference to build mobile apps using Swift.
```

Reply:
@noahmulfinger is correct. The preferred order is: .NET, Kotlin, Swift, Java, Qt.